applications/SwimmingDEMApplication/custom_processes/spatial_dependant_porosity_solution_body_force_process.cpp

```diff
@@ -12,12 +12,12 @@
 //
 
 // System includes
-#include <math.h>
+#define _USE_MATH_DEFINES
+#include <cmath.h>
```

Comment:
please use [this](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/includes/global_variables.h#L36) instead